segment_language.go

```diff
@@ -10,6 +10,20 @@ type language struct {
 	version      string
 }
 
+// DisplayMode represents a value for the property display_mode (always, context or never)
+type DisplayMode string
+
+const (
+	// DisplayModeProperty sets the display mode (always, when_in_context, never)
+	DisplayModeProperty Property = "display_mode"
+	// DisplayModeAlways displays the segement always
+	DisplayModeAlways DisplayMode = "always"
```

Comment:
I was incorrect in my previous example. Just like in `segment_path.go` it's better to use strings:

```golang
// DisplayModeAlways displays the segement always
DisplayModeAlways string = "always"
// DisplayModeContext displays the segment when the current folder contains certain extensions
DisplayModeContext string = "context"
// DisplayModeNever hides the segment
DisplayModeNever string = "never"
```